src/api/lib/tasks/dev/requests.rake

```diff
@@ -227,6 +227,43 @@ namespace :dev do
       end
     end
 
+    desc 'Creates a request that can be forwarded upon acceptance'
+    task forwardable_request: :development_environment do
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      admin = User.default_admin
+      admin.run_as do
+        home_admin_branched_data = branch_package(
+          source_project_name: 'home:Admin',
+          source_package_name: 'hello_world',
```

Comment:
This doesn't exist so it can't be branched.

Reply:
sure but...

Reply:
It was something quick and dirty to have something to test in dev. It will be a bit more work to wire it up so we can have it automatically...

Reply:
well, don't commit quick and dirty things...

Reply:
I've added it to the automated data generation

Reply:
You mean to the `data:test_data` task? That's nice but it *still* should work independently of it...

Reply:
I added some bits to make sure the project and package exists